src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,49 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        %ul.list-unstyled
+          - if bs_request.accept_at.present?
+            %li
+              %i.fas.fa-exclamation-circle.text-danger
+              - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+                Auto-accept was set to
+                %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
+                = succeed '.' do
+                  = l bs_request.accept_at, format: :only_date
+              - elsif bs_request.accept_at.past?
+                This request will be automatically accepted when it enters the 'new' state.
+              - else
+                This request will be automatically accepted in #{time_ago_in_words(bs_request.accept_at)}.
+          - grouped_open_reviews = BsRequest::ReviewsOverview.new(bs_request).grouped_open_reviews
+          - if bs_request.accepted_reviews.present? || bs_request.declined_reviews.present? || grouped_open_reviews.present?
+            .row
+              .col-auto.pr-0
+                %h3
```

Comment:
To have a proper [typographic hierarchy](https://www.toptal.com/designers/typography/typographic-hierarchy), the reviews section should have a smaller heading to not overshadow the request heading (the one at the top of this page).

Reply:
done